### ISSUE TITLE
hi, is there one issue about the calculation for the info_nce_loss during the warm up stage? 

### ISSUE THREAD

**Issue by LanXiaoPang613** [lxysl/PLReMix#2]
![image](https://github.com/lxysl/PLReMix/assets/35187545/599a67d7-13c6-4ef0-b583-027986c18271)
The issue is shown in the figure. The loss_crl is alaways 1 during training. The same problem occurs in the calculation of PLR loss which is depicted as following:

**Comment by LanXiaoPang613** [comment]
![image](https://github.com/lxysl/PLReMix/assets/35187545/79f63d4d-7a28-4171-80a8-1becfbc0da3e)

**Comment by LanXiaoPang613** [comment]
![image](https://github.com/lxysl/PLReMix/assets/35187545/299eb365-e56c-44bb-8b94-103d17f16a8f)
In FlatNCE, it also calculates the CE loss between the logits and pesudo labels. But in this paper, when using only the first terms, i think the parameters of the projector cannot be updated.

**Comment by lxysl** [comment]
please refer to this post: https://github.com/Junya-Chen/FlatCLR/issues/2#issuecomment-926294410
